src/core/types.ts

```diff
@@ -65,5 +70,5 @@ export type LT = ({ lt: string } & { lte?: never }) | ({ lt?: never } & { lte: s
 export type RangeFilter = (GT | LT) & Partial<GT> & Partial<LT>;
 
 export type Filter = {
-  [property: string]: EqualFilter | OneOfFilter | RangeFilter
+  [property: string]: DwnInterfaceFilter | DwnMethodFilter | EqualFilter | OneOfFilter | RangeFilter
```

Comment:
Sanity: is this really necessary? IMO this is much more confusing than what it was before.

Reply:
Removed